acceptance/openstack/identity/v3/limits_test.go

```diff
@@ -38,3 +40,57 @@ func TestLimitsList(t *testing.T) {
 	_, err = limits.ExtractLimits(allPages)
 	th.AssertNoErr(t, err)
 }
+
+func TestCreateLimits(t *testing.T) {
+	err := os.Setenv("OS_SYSTEM_SCOPE", "all")
+	th.AssertNoErr(t, err)
+	defer os.Unsetenv("OS_SYSTEM_SCOPE")
+
+	limitDescription := tools.RandomString("TESTLIMITS-DESC-", 8)
+	resourceLimit := tools.RandomInt(1, 100)
+	resourceName := "image_size_total"
+
+	clients.RequireAdmin(t)
+
+	client, err := clients.NewIdentityV3Client()
+	th.AssertNoErr(t, err)
+
+	project, err := CreateProject(t, client, nil)
+	th.AssertNoErr(t, err)
+
+	// Find image service (glance on Devstack) which has precreated registered limits.
+	// TODO: Use registered limits API to create global limit and then overwrite it with limit.
```

Comment:
I'm fine skipping these tests for versions of openstack for which our workaround doesn't work. Just make sure that we log an issue for implementing the required registered limits API calls, and we reference it here in a comment.

You can skip the tests with something like:
```go
clients.SkipReleasesBelow(t, "stable/xena")
```